docs/release-notes/release-notes-0.19.0.md

```diff
@@ -439,6 +439,17 @@ The underlying functionality between those two options remain the same.
   compatibility.
 
 ## Breaking Changes
+
+* ⚠️ **Breaking Change (lncli only)**: The `lncli listchannels` command output has been
+  updated to fix a misnomer in the field names. The field previously named
+  `chan_id` (which actually contained the short channel ID) has been renamed to
+  `scid` to correctly represent its content. The `chan_id` field now contains
+  the actual channel ID as defined in BOLT02. This is a breaking change for any
+  automation scripts that parse the `chan_id` field from the command output.
+  Please update your scripts to use the new `scid` field name if you were
+  previously using the short channel ID value. For more details,
+  see the [lncli Additions](#lncli-additions) section.
```

Comment:
I know it says "lncli only" at the beginning, but should we still add "This does not affect the RPC response which remains unchanged.", just to be extra clear?

Reply:
also added `lncli closedchannels`